aviary/interface/methods_for_level1.py

```diff
@@ -130,32 +121,7 @@ def run_level_1(
     # else:
     kwargs['optimizer'] = optimizer
 
-    if phase_info is None:
-        if mission_origin == 'GASP':
-            from aviary.interface.default_phase_info.gasp import phase_info, phase_info_parameterization
-            kwargs['phase_info_parameterization'] = phase_info_parameterization
-
-        else:
-            from aviary.interface.default_phase_info.flops import phase_info, phase_info_parameterization
-            kwargs['phase_info_parameterization'] = phase_info_parameterization
-    else:
-        # Load the phase info dynamically from the current working directory
-        phase_info_module_path = os.path.join(os.getcwd(), "outputted_phase_info.py")
```

Comment:
This logic got moved to methods_for_level2.py - since the AviaryProblem is now getting EOM type from reading the input deck, everything related to logic branches based on EOM have to get moved down the chain.